nixos/modules/services/networking/thelounge.nix

```diff
@@ -1,25 +1,24 @@
 { pkgs, lib, config, ... }:
 
 with lib;
-
 let
```

Comment:
Not sure if I agree with this, to be honest. (I'm being told that `nixpkgs-fmt` was used to format the file, but I cannot reproduce the removal of this newline with the web demo, at least, maybe the web demo lags behind the current CLI build?)

Other than that, the formatting changes that were made look good.

(Not trying to discredit the author of this PR or `nixpkgs-fmt`, just wanted to bring this up since it was something I noticed.)